### DISCUSSION TITLE
RSS Feed

### DISCUSSION THREAD

**Discussion by huehnerlady** [google/docsy#640] [Q&A] [answered]
Hi,

I have seen that you do support [RSS feeds](https://www.docsy.dev/docs/adding-content/content/#rss-feeds).
But I cannot find a Button or anything to it on the example page for the blog there to see how it is working.
Is there any trick? From what I have seen in the internet so far the link should be https://example.docsy.dev/blog/index.xml, but even that doesn't work. 

Can someone maybe help me?

**Comment by narrenfrei** [comment]
Take a look on this:
https://www.docsy.dev/docs/adding-content/content/#rss-feeds

Search your config.toml for `disableKinds` and `rss_sections`.
When 'RSS' is not in the list of `disableKinds` you should see an RSS button in the top of your blog section.

**Reply by huehnerlady** [reply]
Hi, thanks for your answer. As written above I cannot even find the rss section on the example page, so as a first step I would like to understand where I can find it on the example page?

Having a look at the [config](https://github.com/google/docsy-example/blob/master/config.toml) it seems not to be disabled so it should be there and I can just not find it?

**Reply by pschrey2** [reply]
On the Example Docsy site I found a RSS feed at https://example.docsy.dev/index.xml (root level)

**Reply by LisaFC** [reply]
Yes, by default the template just creates one RSS feed at root level (we should probably document where it appears).

I'll take a proper look at the feature again, as I think Hugo's RSS functionality has improved since we created the template....

**Reply by LisaFC** [reply]
Will also, now that I've looked at it, investigate why the RSS button has vanished....

**Reply by LisaFC** [reply]
AHA. Figured it out. So the template *should* create site-level *and* section RSS feeds, but we'd accidentally disabled section RSS feeds in the example site in https://github.com/google/docsy-example/pull/85 (if you specify an [outputs] section it overrides the default output formats of HTML and RSS, ah, Hugo is hilarious)

Will update all our example config.tomls

**Reply by LisaFC** [reply]
Added a note in https://github.com/google/docsy/pull/666

**Comment by LisaFC** [comment, marked as answer]
From upthread, as you can't mark a comment as an answer...

> AHA. Figured it out. So the template should create site-level and section RSS feeds, but we'd accidentally disabled section RSS feeds in the example site in google/docsy-example#85 (if you specify an [outputs] section it overrides the default output formats of HTML and RSS, ah, Hugo is hilarious)

Have updated the example config.toml and added a note to the docs in #666

**Reply by huehnerlady** [reply]
cool thank you, will you fix this feature in the example site again? :)

**Reply by LisaFC** [reply]
It's fixed in the example site config now: https://example.docsy.dev/blog/

Let me know if you need anything else!

**Reply by huehnerlady** [reply]
ah I did not find the PR for it, but I see it works now, many thanks :)
